release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4400,6 +4400,28 @@ $ oc adm release info 4.13.51 --pullspecs
 
 * Previously, Open vSwitch (OVS) could function by using older configurations after a cluster upgrade operations. These older configurations might have caused issues for networking components that interact with OVS. With this release, OVS is restarted after an upgrade operation so that OVS is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
 
+[id="ocp-4-13-52-bug-fixes"]
+==== Bug fixes
+
+* Previously, a group ID was not added to the `/etc/group` in the container when the `spec.securityContext.runAsGroup` attribute was set in the pod resource. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-41247[*OCPBUGS-41247*])
+
 [id="ocp-4-13-51-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.52
+[id="ocp-4-13-52"]
+=== RHSA-2024:7939 - {product-title} 4.13.52 bug fix and security updates
+
+Issued: 16 October 2024
+
+{product-title} release 4.13.52, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7939[RHSA-2024:7939] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:7941[RHSA-2024:7941] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.52 --pullspecs
+----
```

Comment:
To mark the end of the 4.13.52 section, add the following sub-section:

[id="ocp-4-13-52-updating"]
==== Updating
To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].